README.md

```diff
@@ -1502,11 +1502,20 @@ Additionally, if a search directory is specified in the form
   Enable the PERUSE MPI data analysis interface.
 
 * `--enable-heterogeneous`:
-  Enable support for running on heterogeneous clusters (e.g., machines
-  with different endian representations).  Heterogeneous support is
-  disabled by default because it imposes a minor performance penalty.
-
-  ***THIS FUNCTIONALITY IS CURRENTLY BROKEN - DO NOT USE***
+  Enable support for running on heterogeneous clusters where data
+  types are equivalent sizes across nodes, but may have differing
+  endian representations.  Heterogeneous support is disabled by
+  default because it imposes a minor performance penalty.
+
+  **NOTE:** Open MPI's heterogeneous support *will not correct for
```

Comment:
Open MPI's heterogeneous support *might* not be working properly in situations where the floating point representation differs significantly between the source and target architectures, especially when the exponent has a different number of bits.